packages/domain-picker/src/domain-categories/index.tsx

```diff
@@ -6,28 +6,27 @@ import { Button, Icon } from '@wordpress/components';
 import classNames from 'classnames';
 import { useI18n } from '@automattic/react-i18n';
 import { useState } from '@wordpress/element';
-import { useSelect } from '@wordpress/data';
-
-/**
- * Internal dependencies
- */
-import { DOMAIN_SUGGESTIONS_STORE } from '../../stores/domain-suggestions';
+import { DomainSuggestions } from '@automattic/data-stores';
```

Comment:
I don't think we'll need this, but if we did let's use `import type` when we're only interested in types, not values. It should help ensure our bundlers aren't confused.

Reply:
Done. https://github.com/Automattic/wp-calypso/pull/42409/commits/8dc117e6c1903a545eb66bb404020500817d5b3a